index.js

```diff
@@ -0,0 +1,55 @@
+const {
+  BaseKonnector,
+  requestFactory,
+  saveFiles,
+  addData
+} = require("cozy-konnector-libs");
+const clear = require("clear");
+const figlet = require("figlet");
+const chalk = require("chalk");
+const googleHelper = require("./google");
+
+module.exports = new BaseKonnector(withFakeFields(start));
+
+function withFakeFields(callback) {
+  return function(fields) {
+    // googleHelper.resetConfigStore();
+    return googleHelper.getTokens().then(callback);
+  };
+}
```

Comment:
I created this HOF to simulate fields recovery. Once installed in `cozy-collect', `fields` will be provided, for now, we need to get `fields` from the dev environment.